src/utils/encryption.js

```diff
@@ -66,22 +73,22 @@ const sideChainGenerate = hash => {
   return sidechain;
 };
 
-const sideChain = hash => sha3_256(forge.util.binary.hex.decode(hash));
+//const sideChain = hash => sha3_256(util.binary.hex.decode(hash));
```

Comment:
@EdmundMai i was getting an error cause the library reference, but i was not sure where that came from